packages/agent-toolkit/src/core/tools/platform-api-tools/workforms-tools/create-form-tool/schema.ts

```diff
@@ -0,0 +1,18 @@
+import { z } from 'zod';
+import { GraphQLDescriptions } from '../workforms.consts';
+import { BoardKind } from '../../../../../monday-graphql/generated/graphql';
+
+export const createFormToolSchema = {
+  destination_workspace_id: z.number().describe(GraphQLDescriptions.form.args.destinationWorkspaceId),
```

Comment:
All IDS should be of type string, I also know in your query its ID right? (or should be)

Reply:
This is following our current mutation which is just a direct proxy for the use_template mutation so we use the same types as them. And they are floats right now not IDs in our mutation. This will be changed to IDs eventually when the use_template makes the change